spec/models/spree/billing_integration_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe 'Deprecated BillingIntegration' do
+  before { Gem::Deprecate.skip, @original = true, Gem::Deprecate.skip }
+  after { Gem::Deprecate.skip = @original }
+
+  it 'returns the an instance scoped to PaymentMethod instead' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.